ovirt/resource_ovirt_vnic.go

```diff
@@ -53,6 +58,10 @@ func resourceOvirtVnicCreate(d *schema.ResourceData, meta interface{}) error {
 		Nic(
 			ovirtsdk4.NewNicBuilder().
 				Name(d.Get("name").(string)).
+				Mac(
+					ovirtsdk4.NewMacBuilder().
+						Address(d.Get("vnic_mac_addr").(string)).
```

Comment:
I see that `vnic_mac_addr` attribute is `optional`. So how about we check if it's defined prior to reading it?